include/ck_tile/ops/epilogue/chainer/epilogue_schedule.hpp

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2025, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck_tile/ops/epilogue/chainer/epilogue_graph.hpp"
+#include "ck_tile/ops/epilogue/chainer/cshuffle_chained_epilogues.hpp"
+
+namespace ck_tile {
+
+/// @brief Pre-built CShuffle epilogue sequences
+/// Provides common epilogue patterns
+template <typename Problem>
+struct CshuffleEpilogueSchedule
+{
+    using Init                         = CShuffleEpilogueStageBase<Problem>;
+    static constexpr index_t NumAccess = Init::SFC::get_num_of_access();
+
+    /// @brief Standard epilogue without scaling
+    /// Sequence: Slice -> Cast -> PrepC -> ApplyD -> Store -> MoveWindows
+    CK_TILE_DEVICE static auto make_base_schedule()
```

Comment:
IMHO shorter name would be sufficient here. 
```suggestion
    CK_TILE_DEVICE static auto make_schedule()
```